ruleset/rules/0803-attack-TA0003.xml

```diff
@@ -18,4 +18,15 @@
+    <rule id="92036" level="6">
+        <if_sid>92035</if_sid>
+        <field name="win.eventdata.authenticationPackageName" type="pcre2">NTLM</field>
```

Comment:
Should we use pcre2 as a default type?

Reply:
Talking with Julian Morales, there are 3 engines, that can work simultaneously to process a rule regex. pcre2 is not set as default for retro-compatibility

Reply:
going forward we want to use pcre2 as default